packages/fast-tooling-react/src/form/custom-controls/control.align.props.ts

```diff
@@ -1,14 +1,10 @@
-/**
- * Align vertical class name contract
- */
-export interface AlignFormControlClassNameContract {
-    alignFormControl: string;
-    alignFormControl_control: string;
-    alignFormControl_controlLabel: string;
-    alignFormControl_controlInputContainer: string;
-    alignFormControl_controlInput__top: string;
-    alignFormControl_controlInput__center: string;
-    alignFormControl_controlInput__bottom: string;
-    alignFormControl_softRemove: string;
-    alignFormControl_softRemoveInput: string;
+import { ListControlConfig } from "../templates";
+
+export enum AlignDirection {
```

Comment:
maybe `VerticalAlignment`?

Reply:
I've changed it to `Alignment` but this (in future) will encompass more than just vertical alignment.